BricksAndHearts.UnitTests/ServiceTests/Admin/AdminServiceTests.cs

```diff
@@ -72,4 +75,169 @@ public void CancelAdminAccessRequest_SetsHasRequestedAdminToFalse_ForCorrectUser
         // Assert
         context.Users.Single(u => u.Id == requestedAdminUser.Id).HasRequestedAdmin.Should().BeFalse();
     }
+
+    [Fact]
+    public void GetAdminLists_GetsListOfCurrentAndPendingAdmins_ForAdminUser()
+    {
+        // Arrange
+        using var context = Fixture.CreateReadContext();
+        var service = new AdminService(context);
+
+        // Act
+        var result = service.GetAdminLists().Result;
+
+        // Assert
+
+        result.Item1.Should().BeOfType<List<UserDbModel>>();
+        result.Item2.Should().BeOfType<List<UserDbModel>>();
+    }

```

Comment:
This doesn't feel very valuable if we're not checking what's in the lists - the type system should already guarantee those assertions!